pages/news.py

```diff
@@ -16,6 +16,119 @@ def markdown_link(self):
 def news_items():
     donate_button = """<a class="btn button-custom btn-custom-two donate-button" href="donate.html">""" + _("Donate today.") + """</a>"""
     return [
+        NewsItem("20160523-htl18attack", _("2016-05-23 - Police department's tracking efforts based on false statistics"),
+_("""
+Documents initially made public by the the Missouri police department
+describe their efforts on tracking Freenet usage.  Using a simple
+scheme, they claim achieving a near zero false positive rate for
+tracking the originator of a download.  While we applaud all public
+documentation on attacks, we have to point out that the claimed
+effectiveness of their attacks is based solely on flawed mathematics.
+In reality, the false positive rate of their method is at least 83%,
+and close to 100% in real world scenarios.
+
+The claimed effectiveness of their attack is based on a false
+assumption about the distribution of the HTL of incoming requests.  HTL (Hops
+to Live) is a number embedded in each request that is usually
+decremented when the request is forwarded, starting at a value of 18
+at the originator of the request, and serves to limit the number of
+hops a request can survive on the network.  As a security precaution,
+the HTL is decremented probabilistically on the first few hops.  In
+their tracking efforts, the Missouri police department assumes that
+when multiple requests for pieces of a file arrive from a single node,
+and all these requests have HTL 18 with no requests at HTL 17 or 16,
+this node must be the originator of the requests, with a false
+positive probability decreasing with the number of requests received
+on from the node.  This would be true if whether the HTL is
+decremented is decided per request, which has historically been the
+case in the early days of Freenet.  The vulnerability they claim to
+exploit has however been addressed in 2008 (last touched in [this
+commit][commithtl18]).  Whether the HTL is decremented is since
+decided per connection, so any probability they claim using the
+aforementioned method is false.
+
+For every connection there is a 50% chance that all the HTL18 requests
+(not only a single one, as would be the case with per-request
+probabilistic decrementing) did not originate from the node from which
+we received them but were forwarded one step. So for 10 connections
+(the lowest number of peers Freenet uses), there are on average 5
```

Comment:
10 peers is the minimum number Freenet will use:

        public static final int MIN_PEERS_FOR_SCALING = 10;

(line 220 in node/OpennetManager.java )